src/installer/test/Microsoft.NET.HostModel.Tests/Microsoft.NET.HostModel.Bundle.Tests/BundlerConsistencyTests.cs

```diff
@@ -79,7 +79,11 @@ public void TestWithDuplicateEntriesFails()
             fileSpecs.Add(new FileSpec(BundleHelper.GetAppPath(fixture), "app.repeat"));
 
             Bundler bundler = new Bundler(hostName, bundleDir.FullName);
-            Assert.Throws<ArgumentException>(() => bundler.GenerateBundle(fileSpecs));
+
+            var argEx = Assert.Throws<ArgumentException>(() => bundler.GenerateBundle(fileSpecs));
+            Assert.Equal("Invalid input specification: Found multiple entries with the same BundleRelativePath:"
```

Comment:
Shouldn't the message contain both `fileSpecs[1]` and `fileSpecs[2]`. 
I think this is why the test is failing.

Reply:
In theory it shouldn't because the `GroupBy` creates one group for each duplicate, thus it's only printed once. Regardless, what tests are failing? I don't see any [here](https://dev.azure.com/dnceng/public/_build/results?buildId=452792&view=ms.vss-test-web.build-test-results-tab)

Reply:
The SelectMany() should print all the matching filespecs; At least, that's the expected behavior (we want to see all the conflicting entries).

Reply:
Here are the failing legs: https://dev.azure.com/dnceng/public/_build/results?buildId=452791

Reply:
Some of them are failing due to Git, and I'm not sure why

Reply:
Yes some legs are having checkout problems. 
So, lets just look at the test failures for now:
```
XUnit : error : Tests failed: /root/runtime/artifacts/TestResults/Debug/Microsoft.NET.HostModel.Bundle.Tests_netcoreapp5.0_x64.html [netcoreapp5.0|x64] [/root/runtime/src/installer/test/Microsoft.NET.HostModel.Tests/Microsoft.NET.HostModel.Bundle.Tests/Microsoft.NET.HostModel.Bundle.Tests.csproj]
```